luigi/scheduler.py

```diff
@@ -587,7 +568,7 @@ def set_status(self, task, new_status, config=None):
                         'disabled for {persist} seconds'.format(
                             failures=task.retry_policy.retry_count,
                             task=task.id,
-                            window=config.disable_window,
+                            window=task.retry_policy.disable_window,
```

Comment:
if i followed your pr summary, this is the change necessary to catch the actual value of `disable_window`? but it will still catch the value provided to the scheduler's config disable_window?

Reply:
@dlstadther on this line we just logging the up-to-date `disalbe_window` value, while the necessary change is getting rid of `task.failures` [here](https://github.com/spotify/luigi/pull/2951/files#diff-ec5c8f6c192cef5a08a5e22aa6273604L319) and using `task.retry_policy.disable_window` (which holds the "correct" value of `disable_window`) instead of `task.failures.window` (which can become stale). 

On the 2nd question: :100: correct, `task.retry_policy.disable_window` equalis a per-task `disable_window` value (if specififed) of the one specified if the scheduler's config.